halyard-cli/src/main/java/com/netflix/spinnaker/halyard/cli/command/v1/ShowCommand.java

```diff
@@ -18,8 +18,11 @@
 
 import com.beust.jcommander.Parameter;
 import com.beust.jcommander.Parameters;
+import com.netflix.spinnaker.halyard.cli.execute.v1.ExecuteShow;
```

Comment:
I'm not sure I understand the modularization here.
You have a command in one class and executor in another. Not exactly sure the distinction, but that's ok. However they are in different packages. Why is the primary organization "horizontal" rather than "vertical"? I'd expect better encapsulation keeping the knowledge of execute localized together. Even with ExecuteShow being an inner class. I dont appreciate any implementation reasons why they might be in different packages.

The implementation is even cross-referencing packages. I'm not sure if that is common in Java or not, but I acyclic dependencies typically raise concerns for me.